packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.styles.ts

```diff
@@ -0,0 +1,244 @@
+import {
+  IPersonaCoinStyleProps,
+  IPersonaCoinStyles,
+  PersonaSize,
+} from './Persona.types';
+import {
+  HighContrastSelector,
+  FontSizes,
+  FontWeights,
+} from '../../Styling';
+
+export const getStyles = (
+  props: IPersonaCoinStyleProps
+): IPersonaCoinStyles => {
+  const {
+    className,
+    size,
+    theme,
+  } = props;
+
+  const { palette } = theme;
+
+  // Persona size conditionals
+  const isSize10 = size === PersonaSize.size10 || size === PersonaSize.tiny;
+  const isSize16 = size === PersonaSize.size16;
+  const isSize24 = size === PersonaSize.size24 || size === PersonaSize.extraExtraSmall;
+  const isSize28 = size === PersonaSize.size28 || size === PersonaSize.extraSmall;
+  const isSize32 = size === PersonaSize.size32;
+  const isSize40 = size === PersonaSize.size40 || size === PersonaSize.small;
+  const isSize48 = size === PersonaSize.size48;
+  const isSize72 = size === PersonaSize.size72 || size === PersonaSize.large;
+  const isSize100 = size === PersonaSize.size100 || size === PersonaSize.extraLarge;
+
+  // Persona Sizes
+  const personaSize16 = '16px';
+  const personaSize24 = '24px';
+  const personaSize28 = '28px';
+  const personaSize32 = '32px';
+  const personaSize40 = '40px';
+  const personaSize48 = '48px';
+  const personaSize72 = '72px';
+  const personaSize100 = '100px';
```

Comment:
These consts are also repeated several places, so they should probably be extracted to a common place as well

Reply:
This one is easy enough as a namespace.